ui/desktop/app/routes/scopes/scope/projects/targets/target.js

```diff
@@ -167,16 +186,16 @@ export default class ScopesScopeProjectsTargetsTargetRoute extends Route {
         session
       );
     } catch (e) {
+      const targetErrors = model.get('errors');
+      targetErrors.add('connect', e.message);
+
+      /* eslint-disable-next-line ember/no-controller-access-in-routes */
+      const controller = this.controllerFor(
```

Comment:
@cameronperera What is the purpose of setting `isConnection` to false within the controller?

I don't think we should disable this eslint rule, perhaps we can look for an alternative? Happy to help